admin/app/controllers/solidus_admin/products_controller.rb

```diff
@@ -3,7 +3,10 @@
 module SolidusAdmin
   class ProductsController < SolidusAdmin::BaseController
     def index
-      @products = Spree::Product.all
+      set_page_and_extract_portion_from(
+        Spree::Product.order(created_at: :desc),
+        per_page: Spree::Config[:admin_products_per_page]
```

Comment:
Thoughts about moving this setting to `SolidusAdmin::Configuration`? We could default it to the value in `Spree::Config` as part of the process for deprecating the one in core.